src/builder/server.js

```diff
@@ -192,12 +193,20 @@ export default (webpackConfig, options = {}) => {
 }
 
 export function getItemOrContinue(collection, req, res, next) {
-  const item = collection.find((item) => item.__url === req.originalUrl)
+  const __url = req.originalUrl
+    .replace(process.env.BASENAME, "/")
```

Comment:
can you use baseUrl here? I would like to avoid env var when possible (and here we are in core code so, does not make sense, will make test more difficult).